bootstrap/web/iframe-adapter.js

```diff
@@ -59,23 +59,35 @@ let messageHandler = function(message) {
             //The following cases should be implemented by the user,
             //therefore users will need an eventListener for "message" events
             case 'windowEvents.minimized':
-                console.log('minimized')
+                let minimized = new CustomEvent('windowEvents', {detail: {event:'minimized'}});
```

Comment:
all of these custom events should start with ZoweZLUX, so that they dont have a chance of conflicting with whatever is also in the iframe

Reply:
For example, ZoweZLUX.windowEvents, ZoweZLUX.sessionEvents